coverage/html/base.css

```diff
@@ -0,0 +1,213 @@
+body, html {
```

Comment:
@Syed-Saad-Qamar we have to ignore the folder `coverage` from commits. Please add to `.gitignore`